app/src/main/java/se/nullable/flickboard/ui/Settings.kt

```diff
@@ -954,6 +955,7 @@ enum class LetterLayerOption(override val label: String, val layout: Layout) : L
     SwedishDE("Swedish (MessagEase, German-style)", SV_DE_MESSAGEASE),
     Ukrainian("Ukrainian (MessagEase)", UK_MESSAGEASE),
     French("French (MessagEase)", FR_MESSAGEASE),
+    FrenchAlt("French (Alternative)", FR_ALT),
```

Comment:
Please try to pick a more descriptive name than "alternative". Try to capture the difference to the other french layout. As a last resort, I would go for something like "French (MessagEase, Crazy-Unicooorn)".

Reply:
That's right.
I'm considering "French (Enhanced MessagEase)" or "French (Supercharged)".
What do you think/prefer?

Reply:
I'd prefer something that doesn't make a value judgement or imply that one option is strictly superior to the other one.

Reply:
Sorry, my intention was definitely not to make a value judgement or imply that it is superior.
I was trying to convey the idea of added characters, since this is factually the case. I am sure some people would actually consider a NOT supercharged layout as a superior layout, since they would prefer less clutter.

What about "French (Enriched)" then?
Or maybe "French (Extended Character Set)" if that is not too long?

I'm afraid there is the same issue? In which case, I don't have other ideas to convey the idea of added characters, and we can just go with "French (Variant)" or "French (MessagEase, Crazy-Unicooorn)" like you suggested.